modules/ocf_ceph/facts.d/libvirt-secret-uuid

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+
+if virsh secret-list | grep client.libvirt; then
```

Comment:
plz check if virsh is installed first, otherwise we get stderr in the puppet logs